\section{Biological Principles}
\subsection{How Bacteria grow, divide and maintain their Shape}

\begin{itemize}
    \item \cite{Amir2014} "Getting into shape: How do rod-like bacteria control their geometry?"
    \item \cite{Lleo1990} "Bacterial cell shape regulation ..."
    \item \cite{Bratton2018} MReB decides shape of bacteria
    \item \cite{Daniel2003} "Two Distinct Ways to Make a Rod-Shaped Cell"
    \item \cite{Gitai2005} MReB is a crystal
    \item \cite{Cooper1991} "New" (1991) understanding of how cell wall forms during division cycle
    \item \cite{Garner2021} "Toward a Mechanistic Understanding of Bacterial Rod Shape Formation and
        Regulation"
    \item \cite{IWAI2002} compound that can induce spherical cells
    \item \cite{Karczmarek2007} more details about transition from rod to sphere
    \item \cite{Bean2009} "Inducing a Low-Affinity State in MreB"
    \item \cite{Ausmees2003} "The Bacterial Cytoskeleton"
    \item \cite{vanTeeffelen2018} "Recent advances in understanding how rod-like bacteria stably
        maintain their cell shapes"
    \item \cite{Si2015} "Bacterial growth and form under mechanical compression" (push on cells;
        make them into flat disk-like structures; analyse growth)
    \item \cite{Zapun2008} "The different shapes of cocci"
    \item \cite{DenBlaauwen2008} Nice paper about the molecular processes which are involved to
        maintain the shape of the rods
\end{itemize}

The last decades have fundamentally challenged and changed our understanding of how bacteria retain
their shape.
Rod-shaped bacteria can grow by extending their circular part, or by inserting new material
at the tip of the rod.
Species such as \ac{ecoli} and ac{bsubtilis}~\cite{Errington2020} fall under the first category
while \ac{spombe} represents the latter.
The material is inserted in small bursts on the nanoscale in forms of patches, bands or
hoops~\cite{DePedro2003}.
Despite their differences in wall-size, \ac{bsubtilis} and \ac{ecoli} follow growth rules which are
comparable with respect to the extension of the rod \cite{Chang2014}.

It was long believed that bacteria lack cytoskeletal filaments but with the end of the last
century, it was discovered that the MreB protein fills this role since it "[..] forms an actin-like
cytoskeleton in bacteria [..]" \cite{Erickson2001} (supplement: [p. 1]).
It can polymerize and form filaments which behave similar to actin microfilaments \cite{Dersch2020}.
Studies of its crystal structure further showed the similarity to actin~\cite{vandenEnt2001} and the
MreB, MreC and MreD proteins were identified as homologues of actin \cite{Lowe2017_lj}.\
\cite{Jones2001} studied the species of different bacterial sub-kingdoms which have the MreB gene.
They concluded: "Many of the organisms are rod shaped, similar to B. subtilis and E. coli.
Organisms with more complex shapes, including curved, filamentous, and helical bacteria, were all
abundantly represented." \cite{Jones2001} (supplement: [p. 7]).
Furthermore \cite{Wachi1987} found that mutants of \ac{ecoli} which create defective MreB proteins will be
spherical instead of their natural rod-shaped form.
Our colleagues from Freiburg have validated many of these observations and further showed that MreB
travels along the axis of elongation with a velocity that depends on the size of the filament
\cite{Olshausen2013}.

\begin{itemize}
    \item z-Ring formation (citations and text)
    \item \textbf{TODO look for other cell-division literature (specific to rods)}
    \item \cite{Koch1995} TODO "A physical basis for the precise location of the division site of rod-shaped bacteria: the Central Stress Model Free"
    \item \cite{Bramkamp2009} TODO "Division site selection in rod-shaped bacteria"
    \item \cite{Lan2007} "Z-ring force and cell shape during division in rod-like bacteria"
    \item \cite{denBlaauwen2018} "Is Longitudinal Division in Rod-Shaped Bacteria a Matter of
    Swapping Axis?"
    \item \cite{Billaudeau2017} "Contrasting mechanisms of growth in two model rod-shaped bacteria"
    \item \cite{Wang2010_2} "Robust Growth of Escherichia coli"
\end{itemize}

\begin{figure}[H]
    \centering
    \includegraphics[width=0.3\textwidth]{example-image-a}
    \includegraphics[width=0.3\textwidth]{example-image-b}
    \includegraphics[width=0.3\textwidth]{example-image-c}
    \caption{TODO:
    (A) Insert material along Rod
    (B) Insert material at tips
    (C) Z-Ring formation and division process
    }
\end{figure}

This raises the question whether a rod shape serves any particular purpose.
It can be argued that the earliest cells consisted of "[..] rods and filaments with cocci being
derived and degenerate forms." \cite{Young2006}.
The importance of shapes on the function of cells is highlighted by~\cite{Takeuchi2005} who grew
\ac{ecoli} in very narrow microchambers and thus permanently altered their shape.
This resulted in modified motion of the cells, some unchanged (helical and short crescent cells),
some dysfunctional (coiled cells).
It is important to note that this experiment controlled only the shape of bacteria without altering
any additional biochemical or genetic functions.

\paragraph{Bending Stiffness}

\begin{itemize}
    \item \cite{Shen2022} "Bending stiffness characterization of Bacillus subtilisâ€™ flagellar
    filament"
    \item \cite{Moreau2018} "collective gliding dynamics of ultra-long filamentous cyanobacteria"
\end{itemize}

As discussed previously, the major contributor which allows rod-shaped bacteria to grow an keep
their shape is MreB.
If left alone, the rod will be straight but external effects can alter the morphology
permanently~\cite{Takeuchi2005}.
Multiple efforts have been made in recent years to measure and analyze the bending properties of
individual bacteria.
\cite{Amir2014_2} designed an experiment similar to \cite{Wang2010,Wang2010_protocol} but instead of
optically trapped beads, a flow was used to apply a uniform force on the rod.
Their method is non-invasive and allows for larger force scales.
Cell-division was suppressed with SulA, thus altering the cellular biochemistry.
Since the cells grow continuously, a straight shape was recovered in every case.
With these methods, they demonstrated that the bacteria "[..] exhibit two fundamentally different
modes of deformations and recover from them" \cite{Amir2014_2}.
These modes are elastic in which the cell recovers immediately and plastic which requires growth to
restore its morphology.
Short pulses of forces resulted in elastic deformations from which the cell recovered immediately.
Plastic changes in morphology were only possible under longer periods of persistent force exertion.
Figure X(B) compares the angle of deflection before and after the snap-back
which occurs after turning off the flow.
The linear fit of 66 experimental results under varying conditions shows that both modes contribute
to the behaviour of the cells.

\begin{figure}[H]
    \centering
    \includegraphics[width=0.45\textwidth]{example-image-a}
    \includegraphics[width=0.45\textwidth]{example-image-b}
    \caption{TODO:
    (A) Cell-cell Interactions
    (B) Elastic and Plastic Deformations of the Rod
    }
\end{figure}

\begin{itemize}
    \item \cite{Bertrand2019} lag-phase in bacterial growth
\end{itemize}

\subsection{Physical Interactions between individual Bacteria and Surfaces}

\begin{itemize}
    \item \cite{BrettFinlay2014} bacteria stick to surfaces
    \item \cite{Shah2022} Some rod-shaped bacteria glide on top of surfaces
\end{itemize}

\textbf{TODO look fore more direct interactions}
\begin{itemize}
    \item is there something like septal-junction cyanos?  %%%used%%%
    \item Quorum sensing  %%%used%%%
    \begin{itemize}
        \item \cite{Boyer2009} "Many factors other than cell density were shown to affect
        N-acyl-homoserine lactones accumulation and interfere with the QS signalling process."
            "This review aims to present all factors interfering with the AHL-mediated signalling
            process, at the levels of signal production, diffusion and perception." %%%used%%%
        \item \cite{MorenoGmez2023} %%%used%%%
        \item \cite{Liu2025} %%%used%%%
    \end{itemize}
    \item bacteria fuse to each other when they are long enough at close distance %%%used%%%
\end{itemize}

A colony of bacteria is known to form biofilms~\cite{Dunne2002}.
This strategy is beneficial for the survival of the collective as it allows it to harness growth
factors and migrate when necessary and also plays an important role in bacterial infections in the
biomedical sector \cite{Ong1999}.
The formation of biofilms is only possible due to the attractive nature of bacterial interactions
towards each other and surfaces \cite{Berne2018}.
To analyze bacterial adhesion to surfaces, \cite{vanLoosdrecht1989} investigated the interactions of
various bacteria with negatively charged polystyrene.
They found that the interaction is characterized by a low Gibbs energy of
$2-3kT$ per cell which can be described by a secondary minimum of the DLVO-theory
\cite{Derjaguin1993,Verwey1947} which is relevant for distances $>=1nm$.
At these distances, the DLVO potential acts as a generic adhesive potential.
Particles which stay in the secondary minimum are held together but even weak interactions are
enough to redisperse them which means that this state is reversible.
When approaching closer distances, more intricate mechanisms come into play in which the
heterogeneity of the materials in question needs to be considered.
\cite{Hori2010} described bacterial adhesion as a "two-phase process including an initial, instantaneous
and reversible physical phase (phase one) and a time-dependent and irreversible molecular and
cellular phase (phase two)".

All these considerations become even more intricate when not considering interactions of bacteria
with surfaces but bacteria with each other.
Gram-negative bacteria (eg. \ac{ecoli}) express outer membrane proteins (adhesins) which allow them
to attach to various hosts \cite{Vaca2019,Beachey1981}.
In addition, the flagella can play an important role in the direct adhesion between cells
\cite{Haiko2013}.
Thus it is not correct to blindly apply the results for adhesion to surfaces to the interactions
between bacteria.

The overall attractive nature of the bacterial forces can be seen in the stress response to positive
and negative compression of a colony in a confined space.
\cite{Trejo2013} studied the formation of macroscopic wrinkles when growing bacteria in a confined
space.
They were able to model the strain and stress of the collection of bacteria which requires that
cells are connected by a (to first order) spring-like force.
The developing wrinkles could then be described by a buckling instability.

\cite{Duvernoy2018} used laser techniques and force microscopy to investigate adhesion of individual
\ac{ecoli} and \ac{paeruginosa} cells.
They found that the adhesion forces of the rods are polar, resulting in mechanical tension which in
turn determines daughter cell arrangement.
They measured the size of the grown bacterial colonies, fitted ellipses to their shapes and found
that adhesion and polar adhesion in particular resulted in a greater difference between the long and
short axis of the fitted ellipse and thus a more oval shape rather than circular (WT).
Furthermore, they measured the critical value $N_(2D\/3D)$ where the colony grows into a second
layer on top of the base layer.
It was shown by \cite{Grant2014} Figure X B), that the point of transition
depends on the rigidity of the surrounding gel and \cite{Duvernoy2018} extended this analysis,
showing that the critical value $N_(2D\/3D)$ depends on the adhesive strength and polarity of the
interaction.
\cite{Grant2014} described the growing bacterial colony as a disc with radius $R$ which is pressed
into the agarose slab.
The authors created a purposely-designed simulation in `C++` in order to model the effects of the
gel on the collection of bacteria.
They approximated individual bacteria as multiple overlapping spheres which were linked by nonlinear
springs such that their dynamics correspond to the Euler-Bernoulli dynamic beam theory
\cite{HAN1999}.
The physical interaction potential between multiple bacteria and the surface was of the form

\begin{equation}
    F(\epsilon) = C E_b (d/2)^(2 - \beta) \epsilon^\beta
\end{equation}

where only repulsive effects were considered and no attractive forces which is justified by the
already dense packing of the bacteria in the experiment.
The value $F$ is the magnitude of the force, $E_b$ is the effective Young modulus of the whole
bacterium, $epsilon$ is the distance of overlap between two spheres and $d$ the diameter of the
spheres.
Their model also accounts for friction between cells which is based on Amonton's laws (against the
direction of movement, proportional to the magnitude of the normal force) \cite{Hutchings2021}.
In order to portray growth, new spheres were inserted and cells divided upon reaching a threshold.
In combining all these aspects, the authors have constructed an agent-based model which was able to
produce the stacking behaviour they set out to describe.

\subsection{Extracellular Reactions \& Signalling}

Nutrient uptake/distribution
\begin{itemize}
    \item \cite{Li2025} citation from van-gogh bundles; also contains subsection about nutrient
        uptake
\end{itemize}


\subsection{Emergent Phenomena and Self-Organization}

\begin{itemize}
    \item \cite{Nagarajan2022} TODO use contents of this review
    \item \cite{Ingham2008} TODO swarming and other nice effects
    \item \cite{Starru} TODO also swarming; also has very nice model for rod-shaped bacteria; I have
        not seen this anywhere else; very nice
        \item \cite{Stewart2005} "Aging and Death in an Organism That Reproduces by Morphologically
        Symmetric Division"
        \item \cite{Jin2020} "Influence of cell interaction forces on growth of bacterial biofilms"
        \item 
\end{itemize}

Although the developing wrinkles mentioned in the preceding section can be characterized as a form
of collective self-organization, they require a confined space and would not be observable
otherwise.
In most cases, rod-shaped bacteria grow in a medium separated enough such that they can freely
extend in space.
\cite{vanGestel2015} explored how \ac{bsubtilis} migrates over a surface by forming multicellular
structures.
The cells organize themselves into "van Gogh bundles" of cells
which are tightly aligned in chains which then form filamentous loops.
This phenomenon occurs at the border of the bacterial colony and the migration is driven by two
phenotypically different cell types \cite{Lpez2010}.
"[The authors] propose that surfactin-producing cells reduce the friction between cells and their
substrate, thereby facilitating matrix-producing cells to form bundles" \cite{vanGestel2015}.
It has been shown that mutants which combine \textit{srfA} and \textit{eps} are able to outperform
the wild-type when it comes to colony expansion \cite{Velicer2009}.
In their final stage, van Gogh bundles consist only of matrix-producing cells although they depend
on surfactin-producing ones during their development.
\cite{vanGestel2015} also constructed an agent-based model to describe the observed effects.
Cells are modeled as straight lines in the shape of a filament by connecting their ends to each
other.
They are initially placed horizontally at the bottom of the simulation domain
In order to update to the next position, three mechanisms are involved: (1) Cell elongation (2) Cell
division (3) Cell turning.
While the first two are rather self-explanatory, in the last step, a new spatial orientation of the
cells is chosen in order to minimize the potential energy

\begin{equation}
    V = (\pi - \alpha_1)^2 + (\pi - \alpha_2)^2
\end{equation}

where $\alpha_1$ and $\alpha_2$ denote the angles to their respective neighbors.
By choosing the new orientation at random with probability $P=1-e^(k(V_c-V_n))$ (where $V_c,V_n$ are
the current and new potential energies), the parameter $k$ symbolizes the bending rigidity.
Their analysis found that a high bending rigidity can play a more important role than the overall
cell size, i.e. the growth rate.
Concerning the complexity of their model, the authors noted:
"We did not aim to accurately model the biophysical details of the growth of van Gogh bundles
(parameterization of such a model would be impossible), but rather to make a simple phenomenological
model to shape our intuition [..]" \cite{vanGestel2015}.
While the authors correctly assessed that the parametrization of such a model with just one
particular experiment would be very difficult, multiple sources combined which treat individual
aspects of each cellular aspect can yield enough information to meaningfully produce results.
\cite{Dong2022} showed that van Gogh bundles aid in the self-healing of bacterial colonies.
They observed that an artificially introduced cut heals better at lower curvatures, which
corresponds to a higher rigidity of the ensemble being placed more towards the outer region of the
triangular-shaped cut.
This work served as a precursor for a very recent study by \cite{Li2025} in which they picked up on
the Agent-Based model.
Here, they investigated the interplay of nutrient depletion and multiple biofilms on the
capacity to form van Gogh bundles.
Their model used linear growth on the cellular level which is only a first-order approximation to
the experimentally observed exponential growth discussed previously.
Furthermore, no forces were calculated between the bacteria but a "shoving algorithm" (collision
detection) moved the agents around such that collisions were avoided.
Based on the approach of \cite{vanGestel2015} they implemented cell-turning with.
But more significantly, they implemented a reaction-diffusion system

\begin{equation}
    \frac{\partial C_s}{\partial t} = D_n \nabla C_s + r_n
\end{equation}

where $\nabla$ is the gradient, $D_n$ the diffusion coefficient and $r_n$ is the nutrient sink or
source.
More precisely, the nutrient sinks (sources) $r_n$ have to be located at the positions of cells.
If we assume a simplification to a point-like interaction with the external environment, we can
write as

\begin{equation}
    \frac{\partial C_s}{\partial t} = D_n \nabla C_s  + \sum\limits_i u_i r_n (x_i) \delta(x-x_i)
\end{equation}

where the sum ranges over all cells $i=1..$ and each cell might have an individual exchange range
$u_i$ located at position $x_i$ which is denoted by the well-known dirac delta-distribution
$\delta(x-x_i)$.
The authors compared a single-biofilm system with a multi-biofilm system and showed that
motile cells can convert into matrix-producing cells in a nutrient depleted system.
The authors show that spores in the case of a nutrient-depleted system

